CMakeLists.txt

```
@@ -99,12 +99,18 @@ add_executable(nigiri-import ${nigiri-import-files})
 target_link_libraries(nigiri-import PRIVATE nigiri boost-program_options ianatzdb-res)
 target_compile_features(nigiri-import PUBLIC cxx_std_23)
 
+# --- BENCHMARK ---
+file(GLOB_RECURSE nigiri-benchmark-files exe/benchmark.cc)
+add_executable(nigiri-benchmark ${nigiri-benchmark-files})
+target_link_libraries(nigiri-benchmark PRIVATE nigiri boost-program_options)
```

```suggestion
target_link_libraries(nigiri-benchmark PRIVATE nigiri boost-program_options ianatzdb-res)
```

Without `ianatzdb-res` it crashes on Windows because of missing time zone files.

---

Fixed, thank you